src/components/accordion/accordion.ts

```diff
@@ -3,19 +3,53 @@
  */
 
 import { html, LitElement } from 'lit';
-import { property, customElement } from 'lit/decorators.js';
+import { property, state, customElement } from 'lit/decorators.js';
 import { AccordionItem } from './accordionItem';

```

Comment:
This shouldn't be imported within the Accordion, since it is technically used externally within the slot. Just export it from the index.ts file along with the main component, and they will both get imported together when you import the index to your Story.